x-pack/plugins/upgrade_assistant/public/application/components/es_deprecations/es_deprecations.tsx

```diff
@@ -42,16 +43,30 @@ const i18nTexts = {
   }),
   pageDescription: i18n.translate('xpack.upgradeAssistant.esDeprecations.pageDescription', {
     defaultMessage:
-      'You must resolve all critical issues before upgrading. Back up recommended. Make sure you have a current snapshot before modifying your configuration or reindexing.',
+      'You must resolve all critical issues before upgrading. Back up recommended. Make sure you have a current snapshot before modifying your configuration or reindexing. ',
   }),
   isLoading: i18n.translate('xpack.upgradeAssistant.esDeprecations.loadingText', {
     defaultMessage: 'Loading deprecation issues…',
   }),
 };
 
+const getBatchReindexLink = (docLinks: DocLinksStart) => {
+  return (
+    <EuiLink href={docLinks.links.upgradeAssistant.batchReindex} target="_blank" external={true}>
+      {i18n.translate('xpack.upgradeAssistant.esDeprecations.batchReindexingDocsLink', {
+        defaultMessage: 'Learn more about Kibana batch reindexing API.',
```

Comment:
IMO It's not really clear why a user would want to use the batch reindexing API here. I updated the copy to help. However, we probably want to move some of this outside of the link text.
```suggestion
        defaultMessage: 'To start multiple reindexing tasks in a single request, use the Kibana batch reindexing API.',
```

Reply:
Thanks for this suggestion, James! I had the same thought.